libs/common/src/lib/navbar/navbar.component.ts

```diff
@@ -69,12 +69,15 @@ export class NavbarComponent implements OnInit, OnDestroy {
   kb = this.sdk.currentKb;
   accountUrl = this.account.pipe(map((account) => this.navigationService.getAccountManageUrl(account!.slug)));
   isAccountManager = this.account.pipe(map((account) => account!.can_manage_account));
-
+  isEnterpriseOrGrowth = this.features.isEnterpriseOrGrowth;
   isEntitiesEnabled = this.features.manageEntities;
   isBillingEnabled = this.features.billing;
-  isSyncEnabled = this.features.sync;
   isSynonymsEnabled = this.features.synonyms;
   isActivityEnabled = this.features.activityLog;
+  isPromptLabEnabled = this.isEnterpriseOrGrowth.pipe(
```

Comment:
Since each permission is now hard-coded in the features service, maybe we could move the `isEntreprise` check in the service.